### ISSUE TITLE
Controllers present though hands aren't.

### ISSUE THREAD

**Issue by Squeyed** [ValveSoftware/steamvr_unity_plugin#326]
I'm currently failing to find a solution for this issue.

The Vive controllers are spawned however the hands are not. Any idea what may be causing this?

Here you can see the prefab is set up to show hands and controllers.
![render model](https://user-images.githubusercontent.com/23362256/53020405-baf19a00-344e-11e9-91a3-2f5d6405b455.JPG)

This is a screenshot of the hierarchy during runtime, it should have the hand as a sibling to the controller.
![runtime](https://user-images.githubusercontent.com/23362256/53020418-bfb64e00-344e-11e9-94f7-abbed09fca0d.JPG)

Using Unity 2018.3.4f1 and SteamVR v2.2.0

Any help would be greatly appreciated!

**Comment by zite** [comment]
Are you seeing any errors or warnings in the console? Have you changed the default skeleton actions or bindings? Unfortunately it looks like there's a bug in the Live View where it isn't displaying the skeleton actions correctly. While I fix that you can check the Input Debugger. If you open the Binding Ui, go into your game (while it's running) and in the lower right hand corner open the Input Debugger you should see Skeleton (Left) and Skeleton (Right) listed. If either of those is Unbound then something is wrong.

![image](https://user-images.githubusercontent.com/198704/53040493-59bdcc80-3436-11e9-921f-73e29f3ca2b1.png)

**Comment by Squeyed** [comment]
Hey @keithbradner, thanks for getting back to me so quickly!

I'm very sorry to have wasted your time but since following your list of questions I've discovered what the issue was. In case it's of any use to you in future I'll explain what happened. I had the "action set" set up as a "mirrored" set, somehow this meant I wasn't able to assign the skeleton in the Binding UI, once I changed that to "per hand" I was able to bind them correctly.
